engineering/maintainers.md

```diff
@@ -0,0 +1,38 @@
+# Project Maintainers
+
+## General Guidelines
+
+* Every GitHub repository must have an assigned maintainer.
+* Maintainer is specified in a MAINTAINER file in the top
+  level directory as follows:
+  `Full Name <email> (@github_handle)`
+* Maintainers are in charge of:
+  * Release Management: tagging, release notes, deciding
+    on versioning, etc.
+  * Code Review pull requests and ensure that issues get
+    handled.
+  * Ensure that good practices are followed and that there
+    is consistency among the code base.
+
+## Code Review
+
+* A maintainer should review every PR but can ask for
+  additional reviewers as needed (e.g. someone else is
+  more knowledgeable about as change or has a say as
+  stakeholder) or completely delegate the review.
+* Non-maintainers at source{d} are discouraged to do code
+  review on PRs they are not asked to as a general practice.
+  However, they are allowed to at their discrection if they
+  think it's appropriate on a specific case.
+* Deadline for each code review is 1 working day.
+* If there is a disagreement among the PR author and multiple
+  reviewers and consensus cannot be reached or [it's not worth to reach](http://bikeshed.org/)
+  the maintainer has the final word.
+* A maintainer can merge his own PRs without code review,
```

Comment:
Not sure we have talked exactly about this. They can only do so in PRs regarding maintainance (typos, tags, Makefile...).